sag_ji4.par

```diff
@@ -1,306 +1,306 @@
-31.470742020	|\	55/54	y	65
-31.958660501	|\'	22528/22113	y'	66
-32.446578982	(|.	375/368	?.	67
-32.934497463	(|	45927/45056	?	68
-33.422415944	(|'	52/51	?'	69
-33.910334425	(|''	51/50	?''	70
-34.398252906	'(|.	252/247	'?.	71
-34.886171387	'(|	50/49	'?	72
-35.118091464	~|).	248/243	p.	72.5
-35.374089868	~|)	49/48	p	73
-35.862008349	~|)'	334611/327680	p'	74
-36.349926830	~|)''	48/47	p''	75
-36.588490003	.(|(.	24057/23552	.q.	75.5
-36.837845311	.(|(	8192/8019	.q	76
-37.325763791	'~|)	535815/524288	'p	77
-37.813682272	/|~	46/45	�	78
-38.061940350	(|(..	1701/1664	q..	78.5
-38.301600753	(|(.	1408/1377	q.	79
-38.789519234	(|(	45/44	q	80
-39.277437715	(|('	1024/1001	q'	81
-39.765356196	~|\	16767/16384	�	82
-40.253274677	.//|.	819/800	.=.	83
-40.741193158	.//|	128/125	.=	84
-41.229111639	.//|'	2240/2187	.='	85
-41.717030120	//|..	459/448	=..	86
-42.204948601	//|.	5373459/5242880	=.	87
-42.692867082	//|	6561/6400	=	88
-43.180785563	//|'	525/512	='	89
-43.668704044	//|''	40/39	=''	90
-44.156622525	'//|.	35721/34816	'=.	91
-44.644541005	'//|	39/38	'=	92
-45.132459486	'//|'	77/75	'='	93
-45.620377967	)//|.	54675/53248	�.	94
-46.108296448	)//|	6656/6480	�	95
-46.596214929	)//|'	131072/127575	�	96
-47.084133410	)//|''	37/36	�''	97
-47.572051891	/|)..	18711/18200	n..	98
-48.059970372	/|).	1053/1024	n.	99
-48.547888853	/|)	36/35	n	100
-49.035807334	/|)'	250/243	n'	101
-49.523725815	(|~	176/171	�	102
-50.011644296	'/|).	34543665/33554432	'n.	103
-50.499562777	'/|)	59049/57344	'n	104
-50.987481258	'/|)'	16875/16384	'n'	105
-51.219540249	./|\	11264/10935	.^	105.5
-51.475399739	./|\'	34/33	.^'	106
-51.963318220	/|\..	640/621	^..	107
-52.451236700	/|\.	567/550	^.	108
-52.939155181	/|\	33/32	^	109
-53.427073662	/|\'	185895/180224	^'	110
-53.914992143	(/|.	65/63	�.	111
-54.402910624	(/|	4096/3969	�	112
-54.890829105	(/|'	32/31	�'	113
-55.095545777	'/|\	1082565/1048576	'^	113.4
-55.378747586	'/|\'	24057/23296	'^'	114
-55.866666067	)/|\.	595/576	%.	115
-56.354584548	)/|\	405/392	%	116
-56.842503029	)/|\'	1323/1280	%'	117
-57.330421510	)/|\''	19683/19040	%''	118
-57.818339991	.(|).	91/88	.@.	119
-58.306258472	.(|)	512/495	.@	120
-58.589460281	|\).	67797/65536	�.	120.6
-58.794176953	|\)	8680203/8388608	�	121
-59.282095434	|\)'	137781/133120	�'	122
-59.770013914	(|).	88/85	@.	123
-60.257932395	(|)	729/704	@	124
-60.745850876	(|)'	7425/7168	@'	125
-61.233769357	(|)''	1358127/1310720	@''	126
-61.721687838	'(|).	72171/69632	'@.	127
-62.209606319	'(|)	23914845/23068672	'@	128
-62.465465809	.(|\.	648/625	.m.	128.5
-62.697524800	.(|\	28/27	.m	129
-63.185443281	.(|\'	16384/15795	.m'	130
-63.673361762	|\\	373977/360448	�	131
-64.161280243	(|\.	531441/512000	m.	132
-64.649198724	(|\	8505/8192	m	133
-65.137117205	(|\'	27/26	m'	134
-65.625035686	(|\''	2925/2816	m''	135
-66.112954167	)|\\..	19683/18944	�..	136
-66.600872648	)|\\.	279006525/268435456	�.	137
-67.088791128	)|\\	885735/851968	�	138
-67.576709609	)|\\'	26/25	�'	139
-68.064628090
+!Lower bound	Symbol	Value	Short	233-EDA

```

Comment:
Updated per the version of this file hosted on the main site.

Reply:
Cool.

Reply:
On second thoughts. What is the provenance of "the version of this file hosted on the main site"? Is that a version of sag_JI4.par that you already checked against George's final JI notation spreadsheet? 

Are we leaving the mina 28, 75, 93 and 140 changes for a later pass? Is this pass just intended to bring the .par files up to the the state of play at the time you joined the project?

Reply:
I have no idea about the origins of [sagittal.org/sag_JI4.par](http://sagittal.org/sag_ji4.par). But this comment was intended to be scoped merely to the header. What I could/should have said to be more clear was: "I didn't notice that the header was wrong until I used an automated tool to compare the version of the file I found hosted on the main site with my version. The version on the main site's header was definitely the correct one, since of course the 4th level of the JI notation is based on 233-EDA, not 47-EDA.

Yes, let's leave the the 28, 75, 93, and 140 changes for a later pass. But by later I mean: within a month. It's just that this PR is ginormous as it is and I always get a bit nervous when pushing new commit atop conversations like this, lest we lose them. I already noticed that some of the links I get sent via email from you require a couple steps to connect with the appropriate conversation, and that's my fault for force-pushing some measly little things I noticed after kicking off the PR that I thought would be simpler to patch than to disclaim. You say you don't trust Git(Hub) yet. Me neither! I've been using it for years and still have gaps in my command of the tool which introduce fear-based decision-making to my workflow.

Reply:
I concur, doctor.